### PR TITLE
add Panda CSS detection

### PR DESCRIPTION
**Pull request by ysm-dev**
https://panda-css.com/

ex
- https://panda-css.com/
- https://vitejsvitelfwyue-nesk--5173--c53ab388.local-credentialless.webcontainer.io
- https://play.panda-css.com/
- https://www.hunterjennings.dev/

### CODE REVIEW

File: src/technologies/p.json [hunk 1]
```diff
@@ -374,6 +374,16 @@
     "scriptSrc": "\\.getpaloma\\.com/",
     "website": "https://www.getpaloma.com"
   },
+  "Panda CSS": {
+    "cats": [
+      66
+    ],
+    "css": "--made-with-panda",
```
Comment (ceopeo) [inline src/technologies/p.json:381]: Is it worth adding technology, there are no results anywhere. Also not tracked in **builtwith** or **similartech**
image

**Review by ceopeo — COMMENTED** [review]
Hey mate, thanks for PR!!!
I couldn't find the specified trace in the provided examples

**Comment by ysm-dev** [comment]
> Hey mate, thanks for PR!!! I couldn't find the specified trace in the provided examples

You can access to root CSS variable with run this code in browser console:

```js
window.getComputedStyle(document.documentElement).getPropertyValue('--made-with-panda')
```

**Comment by ceopeo** [comment]
> > Hey mate, thanks for PR!!! I couldn't find the specified trace in the provided examples
> 
> You can access to root CSS variable with run this code in browser console:
> 
> ```js
> window.getComputedStyle(document.documentElement).getPropertyValue('--made-with-panda')
> ```

Ah ok got it, thanks for the details!!! I tried to test the current code locally and apparently the technology is not defined in the current version. Do you have detect tech  locally when test?

**Comment by ysm-dev** [comment]
> > > Hey mate, thanks for PR!!! I couldn't find the specified trace in the provided examples
> > 
> > 
> > You can access to root CSS variable with run this code in browser console:
> > ```js
> > window.getComputedStyle(document.documentElement).getPropertyValue('--made-with-panda')
> > ```
> 
> Ah ok got it, thanks for the details!!! I tried to test the current code locally and apparently the technology is not defined in the current version. Do you have detect tech locally when test?

hmm.. I just tested but it doesn't detect as well for me.

I don't know why..

**Comment by ceopeo** [comment]
> hmm.. I just tested but it doesn't detect as well for me.
> 
> I don't know why..

We need to try to find other traces, the current option will not give us results.

**Comment by ysm-dev** [comment]
I didn't change anything but
it's working for now: for https://panda-css.com/
image

**Pull request merged**